src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -302,23 +304,9 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     if (!peopleInput) {
       return;
     }
-
+    this.handleFlyout();
```

Comment:
I think you can remove this too, no need to show flyout on focus unless clicked

Reply:
This is necessary since the "focusable" area extends outside of the input itself. If the user clicks, but not directly in the html `<input>`  it'll show a focused state (as the intention of our control) but no flyout.

Reply:
Removing this causes the following scenario: 
![picker](https://user-images.githubusercontent.com/11411686/84073620-03a3ce00-a986-11ea-8e14-b4736ce5aef3.png)

I feel like this is confusing to the user especially if not explicitly marked in the control with ui.

Reply:
Got it. The input box should expand to fit the remaining space IMO - can we do that?